packages/@webex/plugin-cc/src/AgentProfile/types.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  DesktopProfileResponse,
+  ListAuxCodesResponse,
+  ListTeamsResponse,
+  UserResponse,
+} from '../AgentProfileService/types';
+
+/**
+ * Represents the request to a AgentProfile.
+ *
+ * @public
+ */
+export interface AgentProfileRequest {
+  /**
+   * CI user id of the User
+   */
+  ciUserId: string;
+
+  /**
+   * Org id of the User
+   */
+  orgId: string;
+}
+
+/**
+ * Represents the response from a AgentProfile.
+ *
+ * @public
+ */
+export interface AgentProfileResponse {
```

Comment:
I think it might be more appropriate to call this AgentProfile?

Reply:
@sreenara It will create a conflict b/w class name and interface that's why I named this interface as AgentProfileResponse

Reply:
Code Corrected!